contracts/ClientFund.sol

```diff
@@ -79,8 +82,17 @@ contract ClientFund is Ownable, Modifiable, Beneficiary, Benefactor, Authorizabl
         serviceActivationTimeout = 1 weeks;
     }
 
-    function setCurrencyManager(address manager) public onlyOwner {
-        currencyManager = CurrencyManager(manager);
+    function setCurrencyManagerAddress(address newAddress) public onlyOwner notNullAddress(newAddress) {
```

Comment:
I suggest this function signature be changed to `changeCurrencyManager(CurrencyManager newCurrencyManager) public` in order to fit the pattern used in other contracts. Alternatively we could also replace "change" with "set" in function that change auxiliary contract address in the full set contracts.